arch/common/gen_isr_tables.py

```diff
@@ -227,6 +227,9 @@ def main():
     nvec = intlist["num_vectors"]
     offset = intlist["offset"]
 
+    if nvec > 2 ** 15:
```

Comment:
could you use power-of lib function?

Reply:
Do you mean [pow](https://docs.python.org/3/library/functions.html?highlight=power#pow)?

Reply:
Yes, or whatever is available for python.
I am also wondering if the 2^15 was an arbitrary selection or it had some rationale behind

Reply:
It's the highest power-of-two that would catch the problem on my specific hello-world, otherwise completely arbitrary.